routes/zotcourseimport.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const fetch = require('node-fetch');
+const router = express.Router();
+const zotcourseUrl = 'https://zotcourse.appspot.com/schedule/load';
+
+router.get('/:scheduleName', async (req, res) => {
```

Comment:
Just to keep it consistent with the other routes, I think we should name this `/loadUserData'.

Reply:
Also we should make it post request again for consistency but also because we're using `req.body`.